src/main/java/main/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package main;
+
+import core.basesyntax.Circle;
+import core.basesyntax.Figure;
+import core.basesyntax.IsoscelesTrapezoid;
+import core.basesyntax.Rectangle;
+import core.basesyntax.RightTriangle;
+import core.basesyntax.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final int MAX_INT = 25;
```

Comment:
MAX_INT is 2 147 483 647. Think about the more informative name for this variable.